### PR TITLE
Format satellite.dbx

### PR DESCRIPTION
**Pull request by mniewrzal**

### CODE REVIEW

File: satellite/satellitedb/dbx/satellitedb.dbx [hunk 1]
```diff
@@ -3,27 +3,31 @@
...
 model bwagreement (
-	key signature 
+	key signature
 
-	field signature blob 
+	field signature blob
 
 	field data blob
-	
+
```
Comment (egonelbre) [inline satellite/satellitedb/dbx/satellitedb.dbx]: any reason for this empty line?

File: satellite/satellitedb/dbx/satellitedb.dbx [hunk 2]
```diff
@@ -3,27 +3,31 @@
 //--- bwagreement ---//
 
 model bwagreement (
-	key signature 
+	key signature
 
-	field signature blob 
+	field signature blob
```
Comment (egonelbre) [inline satellite/satellitedb/dbx/satellitedb.dbx]: any reason for this empty line?

Comment (mniewrzal) [reply satellite/satellitedb/dbx/satellitedb.dbx]: no, I missed that when I was applying dbx formatter.

File: satellite/satellitedb/dbx/satellitedb.dbx (hunk 1 above)
Comment (mniewrzal) [reply satellite/satellitedb/dbx/satellitedb.dbx]: no, I missed that when I was applying dbx formatter.

**Pull request merged**